asyn/Makefile

```diff
@@ -263,11 +263,25 @@ ifeq ($(DRV_FTDI),YES)
   asyn_SYS_LIBS += usb-1.0
   INC += drvAsynFTDIPort.h
   ifeq ($(DRV_FTDI_USE_LIBFTDI1),YES)
-    asyn_SYS_LIBS += ftdi1
-    USR_CXXFLAGS += -DHAVE_LIBFTDI1
+     USR_CXXFLAGS += -DHAVE_LIBFTDI1
+     ifeq ($(FTDI_LIB),)
```

Comment:
Most other parts of this file use 2-space indentation, please don't change that.